pkg/siftool/add.go

```diff
@@ -255,11 +262,33 @@ func getOptions(dt sif.DataType, fs *pflag.FlagSet) ([]sif.DescriptorInputOpt, e
 		}
 
 		opts = append(opts, sif.OptSBOMMetadata(f))
+	case sif.DataOCIBlob, sif.DataOCIRootIndex:
```

Comment:
Nit: could add a space here, to be consistent with the rest of the `switch`?
```suggestion

	case sif.DataOCIBlob, sif.DataOCIRootIndex:
```